helm/dagger/.changes/unreleased/Breaking-20250411-114148.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Breaking
+body: 'helm: Enable using persistent volume claims in DaemonSet'
```

Comment:
Since this is a breaking change, can we get another line explaining how/where the config has moved?